packages/underscore-redirects/test/astro.test.js

```diff
@@ -29,4 +29,282 @@ describe('Astro', () => {
 
 		assert.equal(_redirects.definitions.length, 2);
 	});
+
+	describe('Supports dynamic routes', () => {
```

Comment:
There's a lot of branching in this file that isn't covered by these tests, but I'm going to leave that for later